pkg/install/v1alpha1/openebs_crds.go

```diff
@@ -190,6 +190,9 @@ spec:
   - JSONPath: .metadata.creationTimestamp
     name: Age
     type: date
+  - JSONPath: .status.capacity
+    name: Capacity
+    type: string
```

Comment:
why string? isn't it supposed to be Quantity?

Reply:
kubernetes will support only `integer, boolean, string, number and date` as data type of `type`